blog/urls.py

```diff
@@ -12,23 +16,28 @@
     # path('comment/<int:pk>/',views.StoriCommentsDetail.as_view(),name="comment-detail"),#view a specific comment
     # path('<int:pk>/comment/',views.StoriCommentCreate.as_view(),name="create-comment"),#specific comment 
 ]
+# blogs/id/comments/id
 router = routers.DefaultRouter()
-router.register('blog',StoriViewset,basename="blog" )
+router.register('blogs', StoriViewset, basename="blogs")
```

Comment:
Well need to have consistent names for our routes, models and views

Reply:
Yeah. Either posts, blogs or stori or mastori